server/routes/routeWorkers/standard.js

```diff
@@ -1,7 +1,7 @@
 const { getIn, isEmpty, firstItem, lastItem } = require('../../utils/functionalHelpers')
 const { getPathFor } = require('../../utils/routes')
 
-module.exports = ({ formConfig, licenceService, sectionName, nomisPushService, config }) => {
+module.exports = ({ formConfig, licenceService, sectionName, nomisPushService = null, config = null }) => {
```

Comment:
this component is created for each route, the last two parameters are optional depending on whether the routes use nomis push or not - seems like this could be tidied up a bit.